addons/website/static/src/scss/website.backend.scss

```diff
@@ -142,41 +143,17 @@
             }
         }
 
-        .o_theme_preview_bottom {
-            h5, h6 {
-                line-height: 16px;
-            }
-        }
-
         .o_button_area {
-            @include o-position-absolute(0, 0, 0, 0);
-            transition: opacity 100ms ease 0s;
-            display: flex;
-            flex-flow: column nowrap;
-            justify-content: center;
-            transform: translate3d(0,0,0);
-            background-image: linear-gradient(0deg, rgba(black, 0.6), rgba(black, 0.3));
-            padding: 10% 20%;
             opacity: 0;
             visibility: hidden;
-
-            > .btn {
-                padding: $btn-padding-y-lg $btn-padding-x-lg;
-            }
-
-            hr {
-                width: 100%;
-            }
         }
 
         .o_theme_preview_top:hover {
-            transition: all .3s ease 0s;
-            transform: translateY(-10px);
-            border-color: darken($o-theme-kanban-gray, 26%);
-            box-shadow: 0 15px 12px -8px rgba(0, 0, 0, .4);
+            transform: scale(1.01) translateY(-2px);
+            box-shadow: $box-shadow;
 
             .o_theme_screenshot {
-                animation: o_theme_screenshot_scroll 4s linear .25s infinite alternate;
+                animation: o_theme_screenshot_scroll 4s linear infinite alternate;
```

Comment:
Well, not sure this animation make sense anymore... 🤔
What's the point of forcing people on trying to "decode" this auto-scrolling image when there is a very comfortable preview just one-click away?

 Maybe we should do like "[]space".
 No more overlay buttons; you click the picture and the preview is opened automatically.
 From there either the user confirm the theme or he goes back.. @pierre-pde ?

Reply:
- Kept the auto scrolling (per @pierre-pde request).
- Removed the overlay+buttons when it isn't an installed theme, direct click to open preview 👍

Reply:
Let's keep the auto scrolling and the flow suits me :)